completion/collcell.cwl

```diff
@@ -16,7 +16,6 @@ norobustcr
 #endif
 
 \collectcell
-\collectcell{cmd}#Sd

```

Comment:
Seems to me `\collectcell{<cmd>}` is the form that should be kept.

The usage of `\collectcell` (in tabular preamble) is always followed by a command
```tex
>{\collectcell\usermacro}c<{\endcollectcell}
```

Reply:
Thanks. Meant to remove the `#d` definition part but you're right, it has a mandatory argument.